sql-aggregates/movie-maniacs.sql

```diff
@@ -0,0 +1,8 @@
+select "c"."firstName",
+       "c"."lastName",
+       sum("p"."amount") as "total"
+  from "customers" as "c"
+  join "payments" as "p" using ("customerId")
+  join "rentals" as "r" using ("rentalId")
```

Comment:
This will work without the `rentals` table `join`

Reply:
Oh! Made it more complicated than it needed to be lol!  Thank you, Scott!